lib/mongo/committees.ts

```diff
@@ -1,11 +1,10 @@
 import { Collection, Db, MongoClient, ObjectId, UpdateResult } from "mongodb";
 import clientPromise from "./mongodb";
-import { commiteeType } from "../types/types";
-import { co } from "@fullcalendar/core/internal-common";
+import { committeeInterviewType } from "../types/types";
 
 let client: MongoClient;
 let db: Db;
-let committees: Collection<commiteeType>;
+let committees: Collection<committeeInterviewType>;
```

Comment:
hvorfor bytta du her?

Reply:
![image](https://github.com/user-attachments/assets/2091b730-8eb2-4b80-a402-b7541044a95c)

Reply:
vet ikke helt om jeg skjønner. commiteeType er vel typen til dokumentene som lagres i "commitees" i databasen?
![image](https://github.com/user-attachments/assets/d6eeaf75-fe09-4088-b9e4-068d7346f1a9)

Reply:
Det er to duplikat typer, commiteeType og committeeInterviewType, der committeeInterviewType er den riktige typen.

Reply:
committeeInterviewType er den riktige typen

Reply:
![image](https://github.com/user-attachments/assets/cb7481dd-a34f-4d7d-a770-834251c9a333)
![image](https://github.com/user-attachments/assets/faf88e02-7672-4030-b5d1-351bc7d79fa6)

Reply:
commiteeType inneholder ikke rom

Reply:
ahh, skjønner

Reply:
kanskje endre navn?  eller ikke.. og message bruker vi vel ikke lenger?

Reply:
Kan kanskje gjøres i en egen branch?

Reply:
sure